Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -417,6 +417,11 @@ public async Task RunRunset(bool doContinueRun = false)
                     WorkSpace.Instance.RunsetExecutor.ProcessRunSetActions(new List<RunSetActionBase.eRunAt> { RunSetActionBase.eRunAt.ExecutionStart, RunSetActionBase.eRunAt.DuringExecution });
                 }
 
+                if(mSelectedExecutionLoggerConfiguration.DataPublishingPhase == ExecutionLoggerConfiguration.eDataPublishingPhase.DuringExecution)
+                {
+                    Runners[0].Centeralized_Logger.RunSetStart(RunSetConfig);
```

Comment:
we said you will create new object of Centeralized_Logger instead calling Runners[0]